### PR TITLE
ctb: Rename event in the L2ToL1MessagePasser

### PR DESCRIPTION
**Pull request by maurelian**
**Description**

The new name MessagePassed, is less confusing since the
name 'WithdrawalInitiated' is used in the L2 Bridge.

**Tests**

Existing tests are updated to match the implementation.

### CODE REVIEW

**Comment by tynes** [comment]
Going to need to deeply grep to get all of the changes required for this one

**Review by smartcontracts — APPROVED** [review]

File: packages/contracts-bedrock/contracts/L2/L2ToL1MessagePasser.sol [hunk 1]
```diff
@@ -51,11 +51,11 @@ contract L2ToL1MessagePasser is Semver {
 
     /**
      * @notice Emitted any time a withdrawal is initiated. An extension to
-     *         WithdrawalInitiated so that the interface is maintained.
+     *         MessagePassed to allow for a 4th indexed argument.
      *
      * @param hash The hash of the withdrawal
      */
-    event WithdrawalInitiatedExtension1(bytes32 indexed hash);
+    event MessagePassedExtension1(bytes32 indexed hash);
```
Comment (maurelian) [inline packages/contracts-bedrock/contracts/L2/L2ToL1MessagePasser.sol:58]: Comment to stop mergify

Comment (tynes) [reply packages/contracts-bedrock/contracts/L2/L2ToL1MessagePasser.sol:58]: I think we can consolidate these events, the workflows are going to involve more getting the event by user who is withdrawing and then pulling the hash from the event

**Review by tynes — APPROVED** [review]

File: packages/contracts-bedrock/contracts/L2/L2ToL1MessagePasser.sol (hunk 1 above)
Comment (tynes) [reply packages/contracts-bedrock/contracts/L2/L2ToL1MessagePasser.sol:58]: I'm open to this via a followup PR

Comment (maurelian) [reply packages/contracts-bedrock/contracts/L2/L2ToL1MessagePasser.sol:58]: yeah, agree that's better

**Pull request merged**